artifacts/definitions/Linux/Events/ExecutableFiles.yaml

```diff
@@ -0,0 +1,49 @@
+name: Linux.Events.ExecutableFiles
+description: |
+  This artifact collects Every new executable file in /tmp or login
+  directory from the Linux kernel.
+
+  This artifact relies on the presence of `auditctl` usually included
+  in the auditd package. On Ubuntu you can install it using:
+
+  ```
+  apt-get install auditd
+  ```
+
+  On SUSE operating systems you can install it using:
+  ```
+  zypper install audit
+  ```
```

Comment:
Do we still depend on the audit package being installed with these changes? Best I can tell, we're listening to the audit multicast socket directly and injecting rules directly without using the audit C libraries, correct?